### ISSUE TITLE
Datepicker page shows undefined-aN-ed

### ISSUE THREAD

**Issue by warpech** [Starcounter/UniformDocs#329]
Datepicker page shows `undefined-aN-ed` instead of the expected date. Could this be connected to renaming the app somehow?

@alshakero pls investigate

Screenshot:

<img width="1499" alt="screen shot 2018-07-03 at 10 08 56" src="https://user-images.githubusercontent.com/566463/42207072-447b52f4-7ea9-11e8-9bba-85a0acf6180f.png">

**Comment by alshakero** [comment]
I digged so deep but I still don't understand the reason. 

I had the same issue in my local `uniform.css` demo, I ran `bower up` and it was fixed. But I lost the old `bower_components` so I don't know what was updated (`bower_components` is gitignored). 

Then, under UnifromDocs, I started updating dependancies one by one to match them with `uniform.css` demo, `polymer`  ✔️, `vaadin-date-picker` ✔️, every `vaadin-*` element ✔️. Still didn't work.

Removed vulcanized polymer 2.6.0 and used normal 2.6.0, didn't work. 

Eventually I copied all uniform.css `bower_components` folder into `UnifromDocs/sys` and still, the issue persists. 

I thought there must be a dep I'm forgetting coming from level1, so I updated Starcounter and the issue was fixed. But I don't know which dep was actually causing it. 

I updated Starcounter at the instance and now https://uniform.starcounter.io/UniformDocs/Datepicker is working.
